<launch>
  <arg name="arduino_port" default="/dev/ttyUSB0" />
  <arg name="lidar_port" default="/dev/ttyUSB1" />
  <include file="$(find service_robot)/launch/minimal.launch"/>
  <node pkg="service_robot" name="vel_listener" type="CmdVel_to_Motor_arduino.py" output="screen"/>  
  <node pkg="service_robot" name="fb_to_odom_node" type="feedback_to_odom_final.py" output="screen"/>  
  <node pkg="rosserial_python" type="serial_node.py" name="serial_node">
    <param name="port" value="$(arg arduino_port)"/>
    <param name="baud" value="57600"/>
  </node>  
  <include file="$(find ydlidar_ros)/launch/X4_for_mapping.launch">
     <arg name="port" value="$(arg lidar_port)"/>
  </include>

    <!-- MAPPING -->
  
 <include file="$(find service_robot)/launch/mapping.launch"/>
 <include file="$(find usb_cam)/launch/usb_cam-test.launch"/>
</launch>
